frontend/src/components/layout/Footer/__snapshots__/Footer.test.tsx.snap

```diff
@@ -1,40 +1,91 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Footer renders correctly 1`] = `
-<nav
-  className="footer navbar navbar-expand navbar-light"
+.c0 {
+  bottom: 0;
+  padding: 10px 0;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-flex-wrap: wrap;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: start;
+  -webkit-justify-content: flex-start;
+  -ms-flex-pack: start;
+  justify-content: flex-start;
+  -webkit-align-items: stretch;
+  -webkit-box-align: stretch;
+  -ms-flex-align: stretch;
+  align-items: stretch;
+  -webkit-align-content: flex-start;
+  -ms-flex-line-pack: start;
+  align-content: flex-start;
+}
+
+.c0 a {
+  display: inline;
+}
+
+.c0 a:link,
+.c0 a:visited {
+  color: #ffffff;
+  font-size: 16px;
+  font-family: 'BCSans',Fallback,sans-serif;
+  padding: 0 10px;
+  border-right: 1px solid #666;
+}
+
+.c0 a:hover,
+.c0 a:focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.c0 div.version {
+  color: #ffffff;
+  font-size: 12px;
+  font-family: 'BCSans',Fallback,sans-serif;
+  padding: 0 0 0 10px;
+  -webkit-box-flex: 2;
+  -webkit-flex-grow: 2;
+  -ms-flex-positive: 2;
+  flex-grow: 2;
+  text-align: right;
+}
+
+<div
+  className="c0"
 >
   <a
-    className="nav-link"
     href="http://www.gov.bc.ca/gov/content/home/disclaimer"
-    onClick={[Function]}
-    onKeyDown={[Function]}
   >
     Disclaimer
   </a>
   <a
-    className="nav-link"
     href="http://www.gov.bc.ca/gov/content/home/privacy"
-    onClick={[Function]}
-    onKeyDown={[Function]}
   >
     Privacy
   </a>
   <a
-    className="nav-link"
     href="http://www.gov.bc.ca/gov/content/home/accessible-government"
-    onClick={[Function]}
-    onKeyDown={[Function]}
   >
     Accessibility
   </a>
   <a
-    className="nav-link"
     href="http://www.gov.bc.ca/gov/content/home/copyright"
-    onClick={[Function]}
-    onKeyDown={[Function]}
   >
     Copyright
   </a>
-</nav>
+  <div
+    className="version"
+    data-testid="version"
+  >
+    vundefined
```

Comment:
For some reason this is undefined which is incorrect as the mock would have returned a value.